.cirrus.yml

```diff
@@ -54,6 +54,7 @@ build_task:
   script:
     - git fetch
     - source cirrus-env BUILD
+    - npm run generate-its && git diff --exit-code # Ensure that the generated ITS are up-to-date
```

Comment:
very cool! I like it. One question, when the changes are intentional; the changes will be committed so **this** `git diff --exit-code` should still pass, only when the changes aren't committed 👍🏻

Reply:
Yes exactly 👍